inst/extdata/demscen_factors.csv

```diff
@@ -1,85 +1,253 @@
-SSP2EU_lowdem,MEA,trn_aviation_intl,2050,0.7
-SSP2EU_lowdem,NEN,trn_aviation_intl,2050,0.7
-SSP2EU_lowdem,NES,trn_aviation_intl,2050,0.7
-SSP2EU_lowdem,OAS,trn_aviation_intl,2050,0.7
-SSP2EU_lowdem,REF,trn_aviation_intl,2050,0.7
-SSP2EU_lowdem,SSA,trn_aviation_intl,2050,0.7
-SSP2EU_lowdem,UKI,trn_aviation_intl,2050,0.7
-SSP2EU_lowdem,USA,trn_aviation_intl,2050,0.7
-SSP2EU_lowdem,CAZ,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,CHA,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,DEU,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,ECE,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,ECS,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,ENC,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,ESC,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,ESW,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,EWN,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,FRA,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,IND,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,JPN,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,LAM,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,MEA,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,NEN,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,NES,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,OAS,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,REF,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,SSA,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,UKI,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,USA,trn_shipping_intl,2050,0.9
-SSP2EU_lowdem,CAZ,trn_freight,2050,0.9
-SSP2EU_lowdem,CHA,trn_freight,2050,0.9
-SSP2EU_lowdem,DEU,trn_freight,2050,0.9
-SSP2EU_lowdem,ECE,trn_freight,2050,0.9
-SSP2EU_lowdem,ECS,trn_freight,2050,0.9
-SSP2EU_lowdem,ENC,trn_freight,2050,0.9
-SSP2EU_lowdem,ESC,trn_freight,2050,0.9
-SSP2EU_lowdem,ESW,trn_freight,2050,0.9
-SSP2EU_lowdem,EWN,trn_freight,2050,0.9
-SSP2EU_lowdem,FRA,trn_freight,2050,0.9
-SSP2EU_lowdem,IND,trn_freight,2050,0.9
-SSP2EU_lowdem,JPN,trn_freight,2050,0.9
-SSP2EU_lowdem,LAM,trn_freight,2050,0.9
-SSP2EU_lowdem,MEA,trn_freight,2050,0.9
-SSP2EU_lowdem,NEN,trn_freight,2050,0.9
-SSP2EU_lowdem,NES,trn_freight,2050,0.9
-SSP2EU_lowdem,OAS,trn_freight,2050,0.9
-SSP2EU_lowdem,REF,trn_freight,2050,0.9
-SSP2EU_lowdem,SSA,trn_freight,2050,0.9
-SSP2EU_lowdem,UKI,trn_freight,2050,0.9
-SSP2EU_lowdem,USA,trn_freight,2050,0.9
-SSP2EU_lowdem,CAZ,trn_pass,2050,0.9
-SSP2EU_lowdem,CHA,trn_pass,2050,0.9
-SSP2EU_lowdem,DEU,trn_pass,2050,0.9
-SSP2EU_lowdem,ECE,trn_pass,2050,0.9
-SSP2EU_lowdem,ECS,trn_pass,2050,0.9
-SSP2EU_lowdem,ENC,trn_pass,2050,0.9
-SSP2EU_lowdem,ESC,trn_pass,2050,0.9
-SSP2EU_lowdem,ESW,trn_pass,2050,0.9
-SSP2EU_lowdem,EWN,trn_pass,2050,0.9
-SSP2EU_lowdem,FRA,trn_pass,2050,0.9
-SSP2EU_lowdem,IND,trn_pass,2050,0.9
-SSP2EU_lowdem,JPN,trn_pass,2050,0.9
-SSP2EU_lowdem,LAM,trn_pass,2050,0.9
-SSP2EU_lowdem,MEA,trn_pass,2050,0.9
-SSP2EU_lowdem,NEN,trn_pass,2050,0.9
-SSP2EU_lowdem,NES,trn_pass,2050,0.9
-SSP2EU_lowdem,OAS,trn_pass,2050,0.9
-SSP2EU_lowdem,REF,trn_pass,2050,0.9
-SSP2EU_lowdem,SSA,trn_pass,2050,0.9
-SSP2EU_lowdem,UKI,trn_pass,2050,0.9
-SSP2EU_lowdem,USA,trn_pass,2050,0.9
+demandScen;region;sector;year;factor
```

Comment:
lowDem is the same as demRedStrong or? In that case lowDem should be removed as an option